core/shared/src/main/scala/com/monovore/decline/CommandApp.scala

```diff
@@ -22,7 +22,7 @@ class CommandApp(command: Command[Unit]) {
   }
 
   def main(args: Array[String]): Unit =
-    command.parse(args) match {
+    command.parse(PlatformApp.ambientArgs getOrElse args) match {
```

Comment:
Hands a little tied here, unfortunately. :disappointed:

This is a `main` method, which is required to take `Array[String]` as a param. ScalaJS learned to recognize regular main methods in the last release, but empirically won't actually pass any arguments to them: it will always pass the empty array. That would make this `CommandApp` sort of useless -- since the whole point is to generate a main that parses args for you -- so we have to go rummaging around in the process state for the args that were actually passed to nodejs.